front/app/components/IdeaListScrollAnchor/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { Box } from '@citizenlab/cl2-component-library';
+
+const IdeaListScrollAnchor = () => {
+  return (
+    <Box
+      position="absolute"
+      mt="-150px"
+      id="ideas-list-scroll-anchor"
+      aria-hidden={true}
+    />
+  );
+};
+
```

Comment:
Since this PR is so small already, just added this in here as well rather than creating new branch/PR.